source/usage-openstack.rst

```diff
@@ -194,6 +194,36 @@ cloud-config data as user data
 * `Official cloud-config examples <https://cloudinit.readthedocs.io/en/latest/topics/examples.html#yaml-examples>`_
 * `Cloud-init example <https://www.zetta.io/en/help/articles-tutorials/customizing-instance-deployment-cloud-init/>`_
 
+
+Creating a snapshot image from running VM
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can create a new image from a snapshot of an existing VM that will allow
+you to easily recover a previous version of your VM.
```

Comment:
It could also be used to provision/clone similar VMs, using it as an image template.

Reply:
Added that.